lib/fastlane/plugin/revenuecat_internal/helper/github_helper.rb

```diff
@@ -8,7 +8,9 @@ module Fastlane
 
   module Helper
     class GitHubHelper
-      SUPPORTED_PR_LABELS = %w[breaking build ci docs feat fix perf revenuecatui refactor style test next_release dependencies phc_dependencies minor].to_set
+      PR_LABEL_PREFIX = 'pr:'
+      BASE_LABELS = %w[breaking build ci docs feat fix perf revenuecatui refactor style test next_release dependencies phc_dependencies minor]
+      SUPPORTED_PR_LABELS = (BASE_LABELS + BASE_LABELS.map { |label| "#{PR_LABEL_PREFIX}#{label}" }).to_set
```

Comment:
Hmm I've been thinking whether it makes sense to just remove support for the old labels now, and just update each SDK to the new version when the labels have been updated... Wdyt?

Reply:
Right. Good point. I'm over complicating things. With the dangerfile we can't do that, but we can for the fastlane plugin